.gitignore

```diff
@@ -1,5 +1,5 @@
+lib-test
 dist
-lib
```

Comment:
Ok will merge and we can continue this conversation in https://github.com/vega/voyager/issues/350